require "better_errors"

BetterErrors.use_pry!
